ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -125,54 +123,6 @@ class DashboardIndex extends PureComponent<Props, State> {
     )
   }
 
-  private handleCreateDashboard = async (): Promise<void> => {
```

Comment:
Woot! Happy to see this extracted into a thunk.

Reply:
:)